assets/xml/objects/object_kin2_obj.xml

```diff
@@ -1,25 +1,29 @@
 ﻿<Root>
+    <!-- 
+        Assets for Ocean Spider House's bombable wall, Skullkid's painting, fireplace grate on the path to the Heart
+        Piece, bookshelves and chests of drawers.
+    -->
     <File Name="object_kin2_obj" Segment="6">
-        <DList Name="object_kin2_obj_DL_000120" Offset="0x120" />
+        <DList Name="gOceanSpiderHouseEmpty1DL" Offset="0x120" />
         <DList Name="gOceanSpiderHouseBombableWallDebrisDL" Offset="0x128" />
         <DList Name="gOceanSpiderHouseBombableWallCrackDL" Offset="0x2C0" />
         <DList Name="gOceanSpiderHouseBombableWallDL" Offset="0x360" />
         <Collision Name="gOceanSpiderHouseBombableWallCol" Offset="0x490" />
-        <DList Name="object_kin2_obj_DL_000650" Offset="0x650" />
-        <DList Name="object_kin2_obj_DL_000658" Offset="0x658" />
-        <Collision Name="object_kin2_obj_Colheader_000798" Offset="0x798" />
-        <DList Name="object_kin2_obj_DL_000820" Offset="0x820" />
+        <DList Name="gOceanSpiderHouseEmpty2DL" Offset="0x650" />
+        <DList Name="gOceanSpiderHouseSkullkidPaintingDL" Offset="0x658" />
+        <Collision Name="gOceanSpiderHouseSkullkidPaintingCol" Offset="0x798" />
+        <DList Name="gOceanSpiderHouseEmpty3DL" Offset="0x820" />
         <DList Name="gOceanSpiderHouseFireplaceGrateDL" Offset="0x828" />
-        <Collision Name="object_kin2_obj_Colheader_000908" Offset="0x908" />
-        <DList Name="object_kin2_obj_DL_000CA0" Offset="0xCA0" />
+        <Collision Name="gOceanSpiderHouseFireplaceGrateCol" Offset="0x908" />
+        <DList Name="gOceanSpiderHouseEmpty4DL" Offset="0xCA0" />
```

Comment:
I'd quite like blank lines between the groups that belong to the same thing, this is quite dense without any.